helpers/apiResponse.js

```diff
@@ -1,49 +1,49 @@
 exports.successResponse = function (res, msg) {
-	var data = {
-		status: 1,
-		message: msg
-	};
-	return res.status(200).json(data);
+    var data = {
+        status: 1,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 8 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		status: 1,
```